packages/mgt-components/src/components/mgt-login/mgt-login.ts

```diff
@@ -213,7 +213,7 @@ export class MgtLogin extends MgtTemplatedComponent {
    */
   public async login(): Promise<void> {
     const provider = Providers.globalProvider;
-    if (!provider.isMultiAccountSupportedAndEnabled && (this.userDetails || !this.fireCustomEvent('loginInitiated'))) {
+    if ((this.userDetails || !this.fireCustomEvent('loginInitiated')) && !provider.isMultiAccountSupportedAndEnabled) {
```

Comment:
Updated because the way it was, the second condition was never hit. 

If the provider wasn't enabled for multi-account (false) the second part of the condition was not executed. Therefore, `loginInitiated` was never called for `mgt-login` in single-account mode. We need this capability to work for both the product, but also to know that an actual login is initiated to support our sign-in capabilities in Storybook.

Reply:
Oh!
Thank you for the clarification. We should probably have a comment in here or refactor it as it's a bit weird having the negation of a void return in a condition

Reply:
The thing is... It's not a void! It's a real boolean! It's a way to intercept and cancel the event... So should we leave it there?

Reply:
I added the comment at least, let me know if you feel it's enough!